src/codegen.cpp

```diff
@@ -4251,7 +4251,7 @@ static jl_cgval_t emit_invoke(jl_codectx_t &ctx, const jl_cgval_t &lival, const
                     bool cache_valid = ctx.use_cache;
                     bool external = false;
                     if (ctx.external_linkage) {
-                       if (jl_object_in_image((jl_value_t*)codeinst)) {
+                       if (0 && jl_object_in_image((jl_value_t*)codeinst)) {
```

Comment:
Did you intend to commit this part?

Reply:
Add a comment or something then? Looks very weird to have that standing freely.

Reply:
Without this code we won't link from package images to the system image. Instead each package image will contain a copy of the native code, so I don't think disabling is an option.

Reply:
A comment here and in the other places doing this would indeed have been nice...